### ISSUE TITLE
Fail to open DB on Windows if used Cyrillic names in the path.

### ISSUE THREAD

**Issue by zztalker** [AlexCovizzi/vscode-sqlite#32]
Try Open DataBase in Explorer - nothing happened - no error, no messages at all.

If I try to query DB 

```
notificationsAlerts.ts:41 Error: unable
t.onDidNotificationChange @ notificationsAlerts.ts:41
(anonymous) @ notificationsAlerts.ts:27
e.fire @ event.ts:140
t.notify @ notifications.ts:113
t.notify @ notificationService.ts:56
e._showMessage @ mainThreadMessageService.ts:84
e._showMessage @ mainThreadMessageService.ts:45
e.$showMessage @ mainThreadMessageService.ts:39
e._doInvokeHandler @ rpcProtocol.ts:322
e._invokeHandler @ rpcProtocol.ts:307
e._receiveRequest @ rpcProtocol.ts:239
e._receiveOneMessage @ rpcProtocol.ts:186
(anonymous) @ rpcProtocol.ts:115
e.fire @ event.ts:140
a @ ipc.net.ts:106
n._socketDataListener @ ipc.net.ts:135
emitOne @ events.js:116
emit @ events.js:211
addChunk @ _stream_readable.js:263
readableAddChunk @ _stream_readable.js:250
Readable.push @ _stream_readable.js:208
onread @ net.js:594
console.ts:136 [Extension Host] [9:16:34 AM][vscode-sqlite][ERROR] Error: Error: unable
t.log @ console.ts:136
t._logExtensionHostMessage @ extensionHost.ts:412
(anonymous) @ extensionHost.ts:224
emitTwo @ events.js:126
emit @ events.js:214
emit @ internal/child_process.js:772
_combinedTickCallback @ internal/process/next_tick.js:141
_tickCallback @ internal/process/next_tick.js:180
```

The workaround is to move DB in a folder in which path not contains Cyrillic characters.

**Comment by AlexCovizzi** [comment]
I'll quote what i responded in another issue similar to this:

> I tried different things, but it looks like the sqlite executable on Windows does not support unicode characters, so opening a database from a path with unicode chars, or executing a query with unicode chars doesn't work (you can try it yourself by opening the database using the sqlite executable in the extension's bin directory).

**Comment by AlexCovizzi** [comment]
Fixed in version 0.5.0
